### PR TITLE
Abnormal Security 1.1.5

### PR DESCRIPTION
**Pull request by tyng94**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
- nil -

## Description
- Bug fixes for command `!abnormal-security-get-employee-identity-analysis`
- Deprecate command `!abnormal-security-submit-inquiry-to-request-a-report-on-misjudgement`
- Added new commands `!abnormal-security-submit-false-positive-report` and `!abnormal-security-submit-false-negative-report`
- Updated Abnormal Security Image

## Screenshots
Screenshot 2022-07-22 at 12 06 45 AM
Screenshot 2022-07-22 at 12 07 13 AM
Screenshot 2022-07-22 at 12 07 25 AM
Screenshot 2022-07-22 at 12 07 44 AM

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 1]
```diff
@@ -345,7 +367,7 @@ def get_employee_identity_analysis_genome_data_command(client, args):
     headers = ['description', 'key', 'name', 'values']
 
     markdown = tableToMarkdown(
-        f"Analysis of {email_address}", response.get('histograms', []), headers=headers, removeNull=True)
+        f"Analysis of {email_address}", response.get('data', []), headers=headers, removeNull=True)
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:370]: Why did you change it? Has the API changed?

**Review by MosheEichler — CHANGES_REQUESTED** [review]
Hi @[tyng94](https://github.com/tyng94).
You did an Excellent Job!! :)

Please see the comment

Please notify me when you are done so we can schedule a Demo session before merging the PR.

Thank you.

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py (hunk 1 above)
Comment (tyng94) [reply Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:370]: @MosheEichler the API did not change. the response always returned with field name `data` and we are fixing this bug with the xsoar logic.

**Comment by tyng94** [comment]
@MosheEichler regarding the changes to github/workfllow/ npm and poetry files, when i fetched the upstream files into the abnormalsecurity fork i got a bunch of automatic PRs from dependabot. should i revert those merges?

**Comment by tyng94** [comment]
otherwise, I should be ready for a demo

**Comment by MosheEichler** [comment]
> @MosheEichler regarding the changes to github/workfllow/ npm and poetry files, when i fetched the upstream files into the abnormalsecurity fork i got a bunch of automatic PRs from dependabot. should i revert those merges?

Yes. Please revert all of the changes in the github/workfllow directory and the npm and poetry files leave just the changes under the directory Packs/AbnormalSecurity.
Thanks!

**Comment by tyng94** [comment]
@MosheEichler reverted those changes!

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/README.md [hunk 2]
```diff
@@ -1061,3 +1061,93 @@ Submit an Inquiry to request a report on misjudgement by Abnormal Security
...
+
+#### Context Output
+
+| **Path**                                          | **Type** | **Description** |
+|---------------------------------------------------| --- | --- |
+| AbnormalSecurity.SubmitFalsePositiveReport.detail | String | Confirmation of inquiry sent | 
+
+
+#### Command Example
+```!abnormal-security-submit-false-positive-report recipient_email=abc@def.com sender_email=def@def.com subject=hello```
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/README.md:1134]:
```suggestion
```!abnormal-security-submit-false-positive-report portal_link=https://portal.abnormalsecurity.com/home/threat-center/remediation-history/123455667```
```

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 3]
```diff
@@ -483,6 +505,34 @@ def submit_an_inquiry_to_request_a_report_on_misjudgement_by_abnormal_security_c
...
 
 
+def submit_false_negative_report_command(client, args):
+    recipient_email = str(args.get('recipient_email;', ''))
+    sender_email = str(args.get('sender_email', ''))
+    subject = str(args.get('subject', ''))
+    response = client.submit_false_negative_report_request(recipient_email, sender_email, subject)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.SubmitFalseNegativeReport',
+        outputs_key_field='',
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:515]:
```suggestion
```

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 4]
```diff
@@ -483,6 +505,34 @@ def submit_an_inquiry_to_request_a_report_on_misjudgement_by_abnormal_security_c
...
+
+    return command_results
+
+
+def submit_false_positive_report_command(client, args):
+    portal_link = str(args.get('portal_link;', ''))
+    response = client.submit_false_positive_report_request(portal_link)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.SubmitFalsePositiveReport',
+        outputs_key_field='',
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:528]:
```suggestion
```

**Review by MosheEichler — CHANGES_REQUESTED** [review]
Please run [demisto-sdk format](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/format/README.md) -i Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml

**Comment by MosheEichler** [comment]
A few ore changes and we are ready to go!

**Comment by MosheEichler** [comment]
@tyng94 Please sign the license [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=20181)

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py (hunk 4 above)
Comment (tyng94) [reply Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py]: modified in pr: https://github.com/demisto/content/pull/20181/commits/2a60ec5d18ed2c81a381c97420b1d39339106f51

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py (hunk 3 above)
Comment (tyng94) [reply Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py]: modified in pr: https://github.com/demisto/content/pull/20181/commits/2a60ec5d18ed2c81a381c97420b1d39339106f51

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/README.md (hunk 2 above)
Comment (tyng94) [reply Packs/AbnormalSecurity/Integrations/AbnormalSecurity/README.md]: modified in pr: https://github.com/demisto/content/pull/20181/commits/2a60ec5d18ed2c81a381c97420b1d39339106f51

**Comment by tyng94** [comment]
> @tyng94 Please sign the license [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=20181)

signed!

**Review by MosheEichler — APPROVED** [review]

**Comment by MosheEichler** [comment]
@tyng94 Hi lets schedule a demo please choose a time from [here](https://calendar.app.google/RPmhpf1tN2ZpxDoj7)

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml [hunk 5]
```diff
@@ -581,14 +538,53 @@ script:
...
+    - description: Subject of email
+      isArray: false
+      name: subject
+      required: true
+    description: Submit a False Negative Report
+    name: abnormal-security-submit-false-negative-report
+    outputs:
+    - contextPath: AbnormalSecurity.SubmitFalseNegativeReport.detail
+      description: Confirmation of report sent
+      type: String
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml:566]:
```suggestion
```
We don't need the ouput please remove.

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml [hunk 6]
```diff
@@ -581,14 +538,53 @@ script:
...
+    - description: URL link of threat log in abnormal security portal
+      isArray: false
+      name: portal_link
+      required: true
+    description: Submit a False Positive Report
+    name: abnormal-security-submit-false-positive-report
+    outputs:
+    - contextPath: AbnormalSecurity.SubmitFalsePositiveReport.detail
+      description: Confirmation of report sent
+      type: String
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml:578]:
```suggestion
```
We don't need the output please remove.

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 7]
```diff
@@ -483,6 +505,32 @@ def submit_an_inquiry_to_request_a_report_on_misjudgement_by_abnormal_security_c
...
 
 
+def submit_false_negative_report_command(client, args):
+    recipient_email = str(args.get('recipient_email;', ''))
+    sender_email = str(args.get('sender_email', ''))
+    subject = str(args.get('subject', ''))
+    response = client.submit_false_negative_report_request(recipient_email, sender_email, subject)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.SubmitFalseNegativeReport',
+        outputs=response,
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:515]:
```suggestion
        readable_output=response,
```

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 8]
```diff
@@ -483,6 +505,32 @@ def submit_an_inquiry_to_request_a_report_on_misjudgement_by_abnormal_security_c
...
+
+    return command_results
+
+
+def submit_false_positive_report_command(client, args):
+    portal_link = str(args.get('portal_link;', ''))
+    response = client.submit_false_positive_report_request(portal_link)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.SubmitFalsePositiveReport',
+        outputs=response,
```
Comment (MosheEichler) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:527]:
```suggestion
        readable_output=response,
```

**Review by MosheEichler — COMMENTED** [review]
@tyng94 As we talked in the Demo there is a few comments Please fix it and let me know when you are ready.
Thanks!

**Comment by tyng94** [comment]
@MosheEichler apologies that took longer than expected. i have booked an appointment for tomorrow 1700hrs (gmt+8)

**Comment by tyng94** [comment]
@MosheEichler is there anything we can do to merge this PR? do i need to close this and reopen with the same branch instead?
